sbi/inference/snle/snle_base.py

```diff
@@ -212,8 +198,10 @@ def _train(
 
         # Starting index for the training set (1 = discard round-0 samples).
         start_idx = int(discard_prior_samples and round_ > 0)
+        theta, x, _ = self._get_from_round_bank(start_idx, exclude_invalid_x)
+
         # Get total number of training examples.
-        num_examples = sum(len(theta) for theta in self._theta_bank)
+        num_examples = len(theta)
```

Comment:
previously this was taking the len of every theta in the bank, i.e., for every round it seems to me. now it is taking the length of just the current round, no?

Reply:
Resolved.